Assets/Scripts/Calendar UI.cs

```diff
@@ -80,46 +102,114 @@ private void HandleBackwardClick()
         SetNumberOfBlanks();
         PopulateCalendarGrid();
     }
+    private void HandleCellImageSelect(Cell cell)
+    {
+        foreach (DayCell c in populationList)
+        {
+            if (c != cell)
+            {
+                c.DeSelect();
+            }
+        }
+    }
+    private void HandleSelectCell(Cell cell)
+    {
+        if (pop == null)
+        {
+
+            selectedCell = cell as DayCell;
+
+            pop = Instantiate(popUpPrefab, canvas);
+            pop.MakeButton("Sticker", HandleAddStickerPressed, false);
```

Comment:
const these strings. Might seem unimportant but when you get into the habit of it you never forget to rename a string somewhere. Ie if you have 

```
124: pop.MakeButton("Notes");
125: pop.MakeButton("Sticker");

....

643: pop.MakeButton("Notes");
644: pop.MakeButton("Poop");
645: pop.MakeButton("Pill");
```

and you want to rename "Notes" to "♦+" everywhere, you only have to make 1 change to the const, and you don't risk missing the change at line 643

Reply:
Resolved